src/main/flight/pid.c

```diff
@@ -216,6 +219,18 @@ void pidInitFilters(const pidProfile_t *pidProfile)
     } else {
         dtermNotchFilterApplyFn = nullFilterApply;
     }
+    
+    //extra Dterm PT1 filter
+	static pt1Filter_t ExtraPt1FilterD;
```

Comment:
remove 'extra',

will we ever allow others than pt1?

Reply:
I struggled with the name for this.  At the moment it's an additional Dterm filter, it is always a PT1, and the user can only enable/set the cutoff point.  Hence 'extra' (additional, but fewer letters), Dterm, PT1, Filter.

Happy for any particular name.

Reply:
It may be possible to refactor the new gyroInitLowpassLpf() function written to support dual/multi-stage dterm/yaw lowpass of varying type, we can do further renaming there after this is tested/merged.

for now, I suggest, e.g.:
```
static filter_t *dtermLowpassState
static pt1Filter_t dtermLowpass2State
```

`Extra` is fine for now as well if we want to revisit.

Reply:
Hmm it has to be different from the other names for the same thing